fluxc/src/main/java/org/wordpress/android/fluxc/store/QuickStartStore.kt

```diff
@@ -35,6 +45,32 @@ constructor(private val quickStartSqlUtils: QuickStartSqlUtils, dispatcher: Disp
 
                 return CHOOSE_THEME
             }
+
+            fun getTasksByType(taskType: QuickStartTaskType): List<QuickStartTask> {
+                return QuickStartTask.values().filter { it.taskType == taskType }
+            }
+        }
+    }
+
+    enum class QuickStartTaskType(private val string: String) {
+        CUSTOMIZE("customize"),
+        GROWTH("growth"),
```

Comment:
```suggestion
        GROW("grow"),
```
Can we rename this type `GROW`?  That way we can use verbs for all types to mimic the type labels (i.e. Customize Your Site and Grow Your Audience)?  I made the mistake of mixing nouns and verbs in the [issue](https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/988).  Sorry about that!

Reply:
No worries! Fixed in [3064cab](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1009/commits/3064cabe07b7d02f8e93f292b5c8a3400e4cd7d2) 👍